commit a70ab2d397bfbdb10ab3ba1f98321763105bc017
Author: Josh Cooper <joshcooper@users.noreply.github.com>
Date:   Tue Feb 7 17:33:09 2023 -0800

    Revert "Allow to build with mingw-clang (#579)"
    
    This reverts commit 8cc8f446f5aac13e107161dffbc15d1ee1a58878.

diff --git a/src/x86/sysv.S b/src/x86/sysv.S
index c7a0fb5..26e7fea 100644
--- a/src/x86/sysv.S
+++ b/src/x86/sysv.S
@@ -56,8 +56,8 @@
 
 /* Handle win32 fastcall name mangling.  */
 #ifdef X86_WIN32
-# define ffi_call_i386		"@ffi_call_i386@8"
-# define ffi_closure_inner	"@ffi_closure_inner@8"
+# define ffi_call_i386		@ffi_call_i386@8
+# define ffi_closure_inner	@ffi_closure_inner@8
 #else
 # define ffi_call_i386		C(ffi_call_i386)
 # define ffi_closure_inner	C(ffi_closure_inner)
